scripts/zones/Bhaflau_Thickets/DefaultActions.lua

```diff
@@ -0,0 +1,5 @@
+local ID = zones[xi.zone.BHAFLAU_THICKETS]
+
+return {
+    ['Mythralline_Wellspring_5'] = { special = ID.text.WELLSPRING },
```

Comment:
Should this be messageSpecial instead of special? First time seeing that used

Reply:
its a simplified version that also works